src/cashier/withdraw.es6

```diff
@@ -39,7 +39,6 @@ class Withdraw {
             height: 400,
             'data-authorized': true,
             close: () => {
-                win.dialog('destroy');
```

Comment:
does this leave us with a memory leak? or how do we make sure that the dialog is being destroyed?

Reply:
every time a dialog  get closed, it actually destroy it and when you click on according button, make a new dialog again.
in the withdraw dialog destroying the dialog occurs before restoring it, so cause an reference issue, so I remove the destroying to prevent this.

Reply:
I also add sanity check for it to make sure it does not lose it's functionality with this change

Reply:
Thanks for the explanation, LGTM 👍🏼

Reply:
:bofa: